torch/lib/c10d/FileStore.cpp

```diff
@@ -259,21 +260,28 @@ std::vector<uint8_t> FileStore::get(const std::string& key) {
     }
     // Always refresh since even though the key exists in the cache,
     // it might be outdated
-    pos_ = refresh(file, pos_, cache_);
-    if (cache_.count(regKey) != 0) {
+    pos_ = refresh(
+        file,
+        pos_,
+        [this](const std::string& key, std::vector<uint8_t>&& value) {
+          insertCache(key, std::move(value));
```

Comment:
Superfluous `std::move`?

Reply:
fwiw, this wasn't superfluous - the compiler gets cranky without the extra std::move sprinkled in.

(Though this isn't needed in the new approach)